marlowe/src/Language/Marlowe/Client.hs

```diff
@@ -428,7 +443,7 @@ getAction :: MarloweSlotRange -> Party -> MarloweData -> PartyAction
 getAction slotRange party MarloweData{marloweContract,marloweState} = let
     env = Environment slotRange
     in case reduceContractUntilQuiescent env marloweState marloweContract of
-        ContractQuiescent _reduced _warnings _payments state contract ->
+        ContractQuiescent _reduced _warnings payments state contract ->
```

Comment:
In line 430 above, the min-ADA for a native token will need to be included along with the role token. Also, it looks like explicit min-ADA will also be necessary with lines for 279, as a work-around for the situation that the ledger constraints apparently make line 279 and 290 into separate UTxOs, without sufficient min-ADA for the role token.